spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -93,19 +93,6 @@ def expect_retryable_error(error_class)
           end
         end
 
-        context 'when not all claims are handed off to MAS' do
```

Comment:
This spec was silently assuming that `rrd_mas_disability_tracking` is enabled; on master, it passes when either

```ruby
before { Flipper.enable(:rrd_mas_disability_tracking) }
```

or

```ruby
expect(Flipper.enabled?(:rrd_mas_disability_tracking)).to be_truthy
```
are added.

Therefore, it makes sense that this PR would've caused it to start failing, and that it should be removed now that this toggle is permanently off.